### PR TITLE
Header

### LINKED ISSUE #31: Implement Header
- [ ] Header based on experience fragment
- [ ] Search button
- [ ] Main navigation with expansion on click
- [ ] Centered logo with link back to homepage.
- [ ] House of Asprey static link
- [ ] Language switcher
- [ ] Buttons for concierge, sign in and minicart.
- [ ] Hamburger menu for mobile
    - [ ] additional navigation items for sign in and language switch (replace buttons) 

Out of Scope:

- [ ] Breadcrumb
- [ ] Search overlay
- [ ] Sign In overlay
- [ ] Concierge overlay
- [ ] Minicart overlay

### PR DESCRIPTION
**Pull request by hannessolo**
Fix #31 

Test URLs:
- Before: https://main--asprey--hlxsites.hlx.page/gb/en
- After: https://header--asprey--hlxsites.hlx.live/gb/en

Out of scope:
- Dropdown for language selection
- Links in tools section (cart, account, concierge)

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -17,18 +17,22 @@
   --clr-white: #fff;
   --clr-black: #000;
   --clr-dark-gray: #453950;
+  --clr-darker-gray: #403B49;
+  --clr-medium-gray: rgb(87 87 87);
```
Comment (herzog31) [inline styles/styles.css:21]:
```suggestion
  --clr-medium-gray: #575757;
```

**Review by Kronnox — APPROVED** [review]

File: blocks/header/header-old.css [hunk 2]
```diff
@@ -0,0 +1,275 @@
+/* header and nav layout */
```
Comment (herzog31) [inline blocks/header/header-old.css:1]: This file is not used anymore, right? So can be deleted.

**Review by herzog31 — COMMENTED** [review]
* Please publish the nav file.
* When clicking on a main menu item, the subnavigation opens. When you click again, it does not close though. (Same for searchbar)
* Please add some spacing below the section headers. 
image
* Mobile navigation moves down compared to the original when navigating into a category.
image

File: blocks/header/header.js [hunk 3]
```diff
@@ -1,99 +1,151 @@
...
  * decorates the header, mainly the nav
  * @param {Element} block The header block element
  */
 export default async function decorate(block) {
+  block.innerHTML = '';
   // fetch nav content
   const navMeta = getMetadata('nav');
   const navPath = navMeta ? new URL(navMeta).pathname : '/nav';
-  const resp = await fetch(`${navPath}.plain.html`);
+  const resp = await fetch(`/${getLanguage()}${navPath}.plain.html`);
```
[the remark this answers is not in the document]
Comment (herzog31) [reply blocks/header/header.js:148]: Yes, that is in the scope of https://github.com/hlxsites/asprey/issues/61 :)

[the remark this answers is not in the document]
Comment (hannessolo) [reply blocks/header/header.js:151]: Already fixed but we can take another look as part of #61

**Review by herzog31 — APPROVED** [review]

**Pull request merged**